openapi/workflow_execution_service.swagger.yaml

```diff
@@ -549,10 +539,11 @@ definitions:
     properties:
       run_id:
         type: string
-        title: workflow run ID
+        title: REQUIRED
```

Comment:
I think you meant to use `required: true`. The title field changes the display of items and how they're accessed by downstream clients.

Reply:
I was going to say that i was just copying what was there, but the new issues explain why that's bad :) Will do.

Reply:
Hmmm, is `required` legal here? Swagger keeps complaining, but I haven't found something definitive in the spec either way

Reply:
I don't think `required: true` is valid for properties anymore. My understanding is that the spec follows JSON Schema:

```
properties:
  required:
    -run_id
  run_id:
    type: string
    description: workflow run ID
```

That is, an array at the top level of `properties`, rather than a field within each specific property definition.

I don't know when this change took effect, but I've seen it in most examples for OpenAPI 2.0 and 3.0.

Reply:
Whoops, I lied. `required` should be on the *same* level as `properties` for object definitions/schemas:

```yaml
type: object
required:
  -run_id
properties:
  run_id:
    type: string
    description: workflow run ID
```

Reply:
@jaeddy thanks for that! You are correct, two different ways depending on if you're in parameters or a JSON schema.

https://github.com/ga4gh/data-object-service-schemas/blob/master/openapi/data_object_service.swagger.yaml#L530

https://github.com/ga4gh/data-object-service-schemas/blob/master/openapi/data_object_service.swagger.yaml#L551